docs/docs/troubleshooting.md

```diff
@@ -0,0 +1,33 @@
+---
+id: troubleshooting
+title: Troubleshooting
+sidebar_title: Troubleshooting
+---
+### Summary
+
+The following page provides suggestions for common errors that may occur during firmware compilation. If the information provided is insufficient to resolve the issue, feel free to seek out help from the [ZMK Discord](https://discord.gg/VJnx9kr).
```

Comment:
We have a redirect URL that we use to make sure it's working, and can be replaced in one spot later, this should be:

```suggestion
The following page provides suggestions for common errors that may occur during firmware compilation. If the information provided is insufficient to resolve the issue, feel free to seek out help from the [ZMK Discord](https://zmkfirmware.dev/community/discord/invite).
```